### PR TITLE
Add instructions about how to bring up e2e test cluster

### PR DESCRIPTION
**Pull request by YangLu1031**
**What type of PR is this?**
 /kind documentation

**What this PR does / why we need it**:
Add instructions about how to bring up e2e test cluster.  `./cluster/kube-up.sh` can bring up a cluster but not work with e2e test script, instead `./hack/e2e-internal/e2e-up.sh` works. kubetest also works, but due to some go module incompatible issues in kubetest, need to do some walkaround to update kubetest binary. Although kubetest can extract different versions of k8s binaries, but for local test, it seems not necessary. So `./hack/e2e-internal/e2e-up.sh` should be the best way.

**Special notes for your reviewer**:
Some discussions about build kubetest: 
https://github.com/Azure/go-autorest/issues/414
https://github.com/Azure/go-autorest/issues/481

**Does this PR introduce a user-facing change?**:

```release-note
Add instructions about how to bring up e2e test cluster
```

### CODE REVIEW

File: cluster/gce/windows/README-GCE-Windows-kube-up.md [hunk 1]
```diff
@@ -95,6 +95,44 @@ To teardown the cluster run:
 PROJECT=${CLOUDSDK_CORE_PROJECT} KUBERNETES_SKIP_CONFIRM=y ./cluster/kube-down.sh
 ```
 
+#### 2b. Create a Kubernetes end-to-end (E2E) test cluster
```
Comment (pjh) [inline cluster/gce/windows/README-GCE-Windows-kube-up.md]: I'm not sure that these headings are accurate:

\#### 2a. Create a regular Kubernetes cluster
\#### 2b. Create a Kubernetes end-to-end (E2E) test cluster

Is there any difference in the resulting clusters? Can e2e tests be run against clusters brought up using the 2a steps?

I'm wondering if this should instead be:

\#### 2. Create a Kubernetes cluster
...explain that there are two ways that work, and explain what the difference between them is...
\##### Using `kube-up`
\##### Using `kubetest`

What do you think?

Comment (YangLu1031) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md]: Agree, did some experiments, the main differences are 
1. Both e2e test clusters and e2e test scripts are setup using the config `cluster/gce/config-test.sh`, so the test scripts look for the e2e cluster only and run tests.
2. Only e2e test clusters enables SSH on windows nodes. Regular clusters are not. So couldn't dump logs thru scp on regular cluster windows node.

Added explanations.

Comment (YangLu1031) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md]: Also the differences between `kubetest  --up` & `./hack/e2e-internal/e2e-up.sh` is that kubetest has extra step at the end: `./cluster/kubectl.sh --match-server-version=false version` which seems just update the kubectl binary. And the kubectl.sh will be deprecated soon from the file descriptions.

**Comment by pjh** [comment]
/lgtm
/approve

Thanks for checking on those things and updating the description.

**Pull request merged**